elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -14,8 +14,18 @@ div.pl-order-blocks-bottom {
 
 li.pl-order-blocks-code {
   list-style-type: none;
+  padding: 0px !important;
 }
 
+li.pl-order-blocks-code .badge{
+    margin: 5px !important;
+}
+
+li.pl-order-blocks-code .mb-2{
+  margin-bottom: 0px !important;
+}
```

Comment:
instead of adding this, why not just edit the rule right below this one so that its `:last_child` instead of `:only_child`?

Reply:
or just get rid of it, I guess

Reply:
Yeah, I was trying things along those lines to get the rule below it to impact the submission panel but I couldn't get it working. Removing the one below has the same impact so I might go ahead and remove it in favor of this rule if that's okay.